terraform/environments/hmpps-oem/locals.tf

```diff
@@ -153,6 +206,7 @@ locals {
         }
       }
     }
+    "account" = module.baseline_presets.ssm_parameters_all.account
```

Comment:
If we set enable_ec2_oracle_enterprise_managed_server = true  then we don't need this.